Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine
+
+RUN mkdir /core
+COPY src/ /core/
+WORKDIR /core
+RUN apk add python3
+RUN pip3 install --upgrade pip
+RUN pip3 install -r requirements.txt
+RUN chmod +x ./*.py
+
+CMD python3 ./start.py
```

Comment:
I know we all forget about it but:
https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Reply:
yeah I keep forgetting about that...